README.md

```diff
@@ -71,19 +85,18 @@ future = protocol.subscribe(topic='topic_name', callback=callback, exception_han
 
 ### Publish a message to topic
 
-```
+```python
 protocol.publish('topic_name', 'Message')
 ```
 
 `protocol.publish` also supports a custom `validation_error_callback`:
 
-```
+```python
 from datetime import datetime
 from socket import gethostname
-from uuid import uuid4
 
 
-def validation_error_callback(invalid_message, exception, protocol):
+def validation_error_callback(invalid_message, exception, proto):
```

Comment:
protocol seems to be the standard reference everywhere. Is there a reason for this change?

Reply:
basically just to highlight that it's different from protocol at the bottom of the snippet, as pycharm was moaning about bad readability - happen to change back